octodns_googlecloud/__init__.py

```diff
@@ -24,7 +24,7 @@ class GoogleCloudProvider(BaseProvider):
 
     CHANGE_LOOP_WAIT = 5
 
-    def __init__(self, id, project=None, credentials_file=None,
+    def __init__(self, id, project=None, credentials_file=None, max_changes=1000,
```

Comment:
This reads as the maximum number of changes the provider will make rather than the batch size.